ui/package.json

```diff
@@ -21,20 +21,24 @@
     "node": ">=6.9 <7.0"
   },
   "dependencies": {
+    "angular2-materialize": "^6.3.0",
```

Comment:
I first attempted to get this to build in devDependencies but it would not load or build properly without them bing in dependencies. There is probably a way to do it in webpack but I was unable to find how.